File: backend/src/models/Employer.js
const mongoose = require('mongoose')

const EmployerSchema = new mongoose.Schema({
  account: {
    type: mongoose.Schema.Types.ObjectId,
    ref: 'Account',
    required: true,
    unique: true
  },
  businessScale: {
    type: String,
    enum: ['Private individuals', 'Companies'],
    required: true
  },
  companyName: {
    type: String,
    required: true
  },
  representativeName: {
    type: String,
    required: true
  },
  position: {
    type: String,
    required: true
  },
  interests: [{ type: String }],
  industry: {
    type: String,
    required: true
  },
  companyInfo: String,
  prove: {
    type: String,
    required: true
  },
  address: {
    type: String,
    required: true
  }
})

module.exports = mongoose.model('Employer', EmployerSchema)

File: backend.NET/Models/Employer.cs
using System;
using System.Collections.Generic;

namespace api.Models;

public partial class Employer
{
    public string EmployerId { get; set; } = null!;

    public string AccountId { get; set; } = null!;

    public string BusinessScale { get; set; } = null!;

    public string? CompanyName { get; set; }

    public string? RepresentativeName { get; set; }

    public string? Position { get; set; }

    public string? Industry { get; set; }

    public string? CompanyInfo { get; set; }

    public string? Prove { get; set; }

    public string? Address { get; set; }

    public virtual Account Account { get; set; } = null!;
}
